shared/common-adapters/tab-bar.js.flow

```diff
@@ -30,7 +30,7 @@ export type ItemProps = Exact<{
 
 export class TabBarItem extends Component<void, ItemProps, void> { }
 
-export type TabBarButtonSource = {type: 'icon', icon: IconType} | {type: 'avatar', avatar: React$Element<*>} | {type: 'nav', icon: IconType}
+export type TabBarButtonSource = {type: 'icon', icon: IconType} | {type: 'avatar'} | {type: 'nav', icon: IconType}
```

Comment:
really this should be `{type: 'avatar', username: string}` and remove username from below